api/core/model_runtime/model_providers/yi/yi.yaml

```diff
@@ -0,0 +1,68 @@
+provider: yi
+label:
+  en_US: 01.AI
+  zh_Hans: 零一万物
+description:
+  en_US: Models provided by 01.AI, such as yi-34b-chat and yi-vl-plus.
+  zh_Hans: 零一万物提供的模型，例如 yi-34b-chat 和 yi-vl-plus。
+icon_small:
+  en_US: icon_s_en.svg
+icon_large:
+  en_US: icon_l_en.svg
+background: "#007057"
```

Comment:
Thanks a ton!
Btw, this color is a bit too eye-catching 👀💥

Suggest going for lighter color values as much as possible, you can refer to JINA's background color for guidance. 😉

Reply:
done